### DISCUSSION TITLE
Hello

### DISCUSSION THREAD

**Discussion by muntik** [home-assistant/frontend#17548] [Theming]
I want to find people with whom I can work in a team and gain experience, I am a beginner and I hope for your support!

**Comment by Quentame** [comment]
You better improve your discussion title, or use the community website instead

**Reply by muntik** [reply]
I just didn’t find anything like that, so I decided to write this in discussions of topics that interest me
Sorry if i'm causing any inconvenience

**Reply by Quentame** [reply]
Of course you can, no worries ! (at least I think)

I meant that with an "Hello" title, that is not attractive and telling the purpose of your idea.

Better try something like "Proposing peer programming support" or "Here to propose my coding skills" ☺️

**Reply by muntik** [reply]
thanks for the suggestion, I want to become better in IT and I need people's help for this in order to gain experience
